variables.tf

```diff
@@ -1,3 +1,15 @@
+variable "availability_zones" {
+  type        = list(string)
+  default     = []
+  description = "The availability zones to place the cache nodes"
+}
+
+variable "at_rest_encryption" {
+  type        = bool
+  default     = false
```

Comment:
Maybe default to `null`, which means you did not set the value. Seems like a good default since its an optional field.